### DISCUSSION TITLE
ScriptReference/Graphics.Blit

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#147] [Notes]
# ScriptReference/Graphics.Blit

https://docs.unity3d.com/ScriptReference/Graphics.Blit.html

**Comment by vertxxyz** [comment]
[`RenderTexture.active`](https://docs.unity3d.com/ScriptReference/RenderTexture-active.html) is set by this function. You may want to cache the active RenderTexture and restore it to maintain the previous state.
